osu.Game.Rulesets.Mania/osu.Game.Rulesets.Mania.csproj

```diff
@@ -15,4 +15,8 @@
   <ItemGroup Label="Project References">
     <ProjectReference Include="..\osu.Game\osu.Game.csproj" />
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="MathNet.Numerics" Version="5.0.0" />
+  </ItemGroup>
```

Comment:
How feasible is it to eliminate this mathnet reference in a way similar to https://github.com/ppy/osu/pull/20963?

Yes I keep banging on about that and it's probably inconvenient but it matters if it's going to decrease binary size / dependency blast radius.

Reply:
Yes, tsunyoku ported the relevant functions to a file. I just have to clean up the implementation